packages/erc3k/contracts/IERC3000Executor.sol

```diff
@@ -8,7 +8,7 @@ pragma experimental ABIEncoderV2;
 import "./ERC3000Data.sol";
 
 abstract contract IERC3000Executor {
-    function exec(ERC3000Data.Action[] memory actions) virtual public returns (bytes[] memory);
+    function exec(ERC3000Data.Action[] memory actions, bytes32 allowFailuresMap, bytes32 memo) virtual public returns (bytes[] memory);
```

Comment:
Should we just call `allowFailuresMap` `failureMap`?

Reply:
No, could be confusing with the failure results bitmap which is logged (and actually should be returned here)